### DISCUSSION TITLE
Known E3SM issue with resubmits on pm-cpu

### DISCUSSION THREAD

**Discussion by JessicaNeedham** [NGEET/fates#1075] [Host Land Model notices]
For FATES users running with E3SM on Perlmutter CPUs, just a heads up that there is a known issue with resubmits not working if short term archiving is on - see [#5901](https://github.com/E3SM-Project/E3SM/issues/5901). A simple work around for now is to turn off short term archiving by setting  ./xmlchange DOUT_S=FALSE.

**Comment by glemieux** [comment]
This issue has been fix with https://github.com/ESMCI/cime/pull/4494
